commons-schemavalidation/src/test/java/eu/europeana/api/commons/CommonsSchemavalidationApplicationTests.java

```diff
@@ -0,0 +1,12 @@
+package eu.europeana.api.commons;
+
+import org.junit.jupiter.api.Test;
+
+
+class CommonsSchemavalidationApplicationTests {
```

Comment:
this test class is doing nothing except context loading. Would be nice top add some tests or remove this